EIPS/eip-000.md

```diff
@@ -149,6 +149,16 @@ transactions using one, it is reasonable for the protocol to only burn the gas c
 As all transactions in the same block pay exactly the same `baseFeePerGas`, the single cost of accessing a cold item is
 divided evenly among all transactions containing such access and the rest of the burned base fee is refunded.
 
+### Setting an absolute minimal cost of cold state access
+
+If a large number of transactions all access the same addresses or slots, the cost of each cold access may get
+way too low which may represent a potential DoS attack vector.
+
+In order to prevent that, the base fee gas component of access cost cannot be lower than `MIN_ACCESS_LIST_ENTRY_COST`
+which is set at equivalent to `WARM_STORAGE_READ_COST` or `100 gas`.
```

Comment:
> It's mixing execution costs with calldata costs. The value may be 100, but we need a better justification for its value.

It is not 100% clear to me if the minimum is even necessary, so this is a draft PR.
Using "calldata cost" as a minimum with compression is 2 bytes which is `32 gas` I think. That is, if we ignore the cost of adding an entry to the `block-level access list`.
Would it makes sense to you if instead of `100 gas` the `MIN_ACCESS_LIST_ENTRY_COST` was set to `32 gas`?

Reply:
@drortirosh why do you see it as mixing?  The rationale as actually that there's no reason a cold-but-amortized SLOAD would cost less than a worm SLOAD.  The latter costs the validator the same or less.  And OTOH we don't want to make warm SLOAD cheaper because it can be called any number of times in one transaction.

@forshtat you're right that if we disregard the above rationale, then 32 gas makes more sense.

And why do you see it as not necessary?  How else do we mitigate an attack where a proposer propagates a valid block that is 16 times bigger than currently possible?